packages/terra-time-input/src/TimeInput.module.scss

```
@@ -92,6 +92,46 @@
     .meridiem-display[disabled] {
       opacity: 1;
     }
+
+    &.is-invalid {
+      background-color: var(--terra-time-input-invalid-background-color, #fff);
+      border: var(--terra-time-input-invalid-border, 1px solid #e50000);
+      box-shadow: var(--terra-time-input-invalid-box-shadow, 0 0 0 1px #e50000);
+
+      &:hover {
+        border: var(--terra-time-input-invalid-hover-border, 1px solid #e50000);
+      }
+
+      &:focus {
+        border: var(--terra-time-input-invalid-focus-border, 1px solid #e50000);
+      }
+
+      &.disabled {
+        background-color: var(--terra-time-input-invalid-disabled-background-color, rgba(111, 116, 119, 0.2));
+        border: var(--terra-time-input-invalid-disabled-border, 1px solid #e50000);
+        color: var(--terra-time-input-invalid-disabled-color, #1c1f21);
+        opacity: var(--terra-time-input-invalid-disabled-opacity, 0.25);
+      }
+    }
+
+    &.is-incomplete {
+      background-color: var(--terra-time-input-incomplete-background-color, #ffc);
+
+      &:hover {
+        background-color: var(--terra-time-input-incomplete-hover-background-color, #ffc);
+      }
+
+      &:focus {
+        background-color: var(--terra-time-input-incomplete-focus-background-color, #ffc);
+      }
+
```

add solid color for desktop initial-focus:
```suggestion
      .desktop.initial-focus {
        background-color: var(--terra-time-input-desktop-incomplete-focused-input-background-color,  rgb(185, 215, 233));
       }
       
```